api-docs/openapi.json

```diff
@@ -141,7 +141,7 @@
     "/api/0/issues/{issue_id}/hashes/": {
       "$ref": "paths/events/issue-hashes.json"
     },
-    "/api/0/issues/{issue_id}/": {
+    "/api/0/issues/{organization_id_or_slug}/{issue_id}/": {
```

Comment:
This should be `/api/0/organization/<organization_id_or_slug>/issues/<issue_id>/`, but maybe we should do this in a separate PR with all of the endpoints that need this update? I feel like these pages need a note at the top about the change

Reply:
oh shit, this shouldn't have been updated, good catch 🙏🏽